core/src/main/kotlin/org/evomaster/core/problem/graphql/schema/FullType.kt

```diff
@@ -0,0 +1,14 @@
+package org.evomaster.core.problem.graphql.schema
+
+/**
+ * FullType: fragment  on __Type
+ */
+data class FullType( var kind: __TypeKind,
+                     var name: String,
+                     var fields : ArrayList<__Field> = ArrayList(),
```

Comment:
when declaring container types, use interfaces and not concrete classes. 
furthermore, in idiomatic Kotlin, when creating empty lists, use either "listOf()" or "mutableListOf()".

this means that can be rewritten into:

var fields : MutableList<__Field> = mutableListOf(),

Note: this does apply to ALL/most other list declarations in this PR

Reply:
Done

Reply:
hi. why is this marked as "Done", while no change is done in the code?